core/src/main/scala/cats/Functor.scala

```diff
@@ -14,38 +14,42 @@ import simulacrum._
 @typeclass trait Functor[F[_]] extends functor.Invariant[F] { self =>
   def map[A, B](fa: F[A])(f: A => B): F[B]
 
-  /** alias for map */
+  /**
+   *  alias for map 
```

Comment:
Because I'm apparently in total nitpick mode over lunch today—there's an extra space before "alias" here. Also both this and the "empty" below should probably be capitalized for consistency, no?